### PR TITLE
redid the when2meet feature til saving to sessions table, it's now sa…

### PR DESCRIPTION
**Pull request by LienLeinad**
…ved as a concatted string with spaces in the middle for easy splitting later on.

-The feature should work the same as my old version until saving to the sessions table, try testing it the same way you did last time and it should work the same. It'll print out the list of strings saved. I'll fix the labeling when we finalize this feature

### CODE REVIEW

File: users/views.py [hunk 1]
```diff
@@ -263,6 +151,11 @@ def home(request):
                 new_hours = request.POST.get('hours')
                 new_subject = request.POST.get('subject')
                 new_end = request.POST.get('end_time')
+                new_sched = ""
+                for s in sched:
+                    new_sched += s + " "
```
Comment (daniddelrio) [inline users/views.py]: You can compress this into one line: 
```python
new_sched = " ".join(sched)
```

**Review by daniddelrio — COMMENTED** [review]
Looks good naman, though my only comment is how the time format looks less neat to look at:
Screen Shot 2020-03-02 at 10 54 57 PM

Other than that and my other comment, you're free to merge

File: users/views.py (hunk 1 above)
Comment (LienLeinad) [reply users/views.py]: I'll fix the labelling issue after the feature is done overall

Comment (daniddelrio) [reply users/views.py]: Great, feel free to merge na after

**Pull request merged**